package.json

```diff
@@ -9,11 +9,14 @@
   "scripts": {
     "test": "jest",
     "test:watch": "jest --watch",
-    "lint": "eslint src",
-    "fmt": "LOG_LEVEL= prettier-eslint --write --no-semi $PWD/'src/**/*.js'",
+    "lint": "eslint src --ext .js,.jsx,.ts,.tsx",
+    "lint:cache": "eslint src --ext .js,.jsx,.ts,.tsx --cache",
+    "fmt": "LOG_LEVEL= prettier-eslint --write --no-semi $PWD/'src/**/*.ts' $PWD/'src/**/*.tsx'",
     "fmt:check": "LOG_LEVEL= prettier-eslint --list-different --no-semi $PWD/'src/**/*.js'",
-    "babel": "babel src --ignore src/*.test.js,src/test_util.js,src/test_setup.js -d dist",
-    "prepack": "npm run babel"
+    "typescript": "rm -rf dist/* && rm -f .tsbuildinfo && tsc",
```

Comment:
I think you want `rm -rf dist` here. The `*` won't get hidden files in the directory.